d2l-tspan.html

```diff
@@ -81,6 +120,30 @@
 				if (this.root.host) {
 					this.root.host.style.height = this.$.float.offsetHeight + 'px';
 				}
+			},
+			focus: function() {
```

Comment:
Note: every HTML element has a native focus method.  For some elements, it does nothing, but for others, it moves the focus to the element.  Whenever we implement a custom `focus` method in a custom-element, it's purpose should be to place the focus, typically on the host element or a focusable element within it's shadow-DOM.  The purpose of this method does not appear to be that, so we should rename this method, since it's effectively overriding the native `focus` method.

Reply:
renamed